sympy/physics/continuum_mechanics/beam.py

```diff
@@ -847,22 +844,21 @@ def _handle_end(self, x, value, start, order, end, type):
         # NOTE : A Taylor series can be used to define the summation of
         # singularity functions that subtract from the load past the end
         # point such that it evaluates to zero past 'end'.
```

Comment:
I would just remove this function. There is no point in having a function that is called in two places with a boolean like this:
```python
def func_A():
    _func(type='A')

def func_B():
    _func(type='B')

def _func(type):
    if type == 'A':
        # A code
    elif type == 'B':
        # B code
```
It would be clear to put the A code and B code in func_A and func_B.